components/kyma-environment-broker/internal/orchestration/handlers/orchestration_handler.go

```diff
@@ -236,6 +236,8 @@ func (h *orchestrationHandler) listOperations(w http.ResponseWriter, r *http.Req
 	var response commonOrchestration.OperationResponseList
 	switch o.Type {
 	case commonOrchestration.UpgradeKymaOrchestration:
+		fmt.Println("orchestrationID is:", orchestrationID)
+		fmt.Println("filter is:", filter)
```

Comment:
I don't think these `Println` statements are a good idea, I would recommend using logger.

Reply:
Hi @wozniakjan I will remove in next version and i would like to make a build image to test resetting totalCount.

Reply:
hi @wozniakjan i fix the println issue and could you help to take a review? thanks.